### DISCUSSION TITLE
Samples

### DISCUSSION THREAD

**Discussion by Bumblebee001** [GrandOrgue/GoOdf#26] [General]
How do I add the samples' path to the respective stops?

**Comment by larspalo** [comment]
Oh, there are many ways to do that. The most basic is to use the "Create new pipes by MIDI nbr from..." button and select the directory to read samples from. But it's possible to do this in other ways too... Just remember to adjust the number of pipes for the rank/stop first (and also make sure to set the first MIDI note correctly if necessary.
